### PR TITLE
Python API

### PR DESCRIPTION
**Pull request by skatedplrn**

### CODE REVIEW

**Comment by skatedplrn** [comment]
Lars, take a look on auth module
I use auth_ prefix for api call, since login_auth looks strange to me
Are you OK to use module prefix in Python API?

File: auger_cli/api/cluster_tasks.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def run_cluster_task(client, wait):
+    from ..projects.api import start_project
+
+    project_id = start_project(client, create_if_not_exist=True)
+    cluster_task = client.config.get_cluster_task()
+
+    create_cluster_task_ex(client, project_id, cluster_task.get(
+        'name'), cluster_task.get('params'), wait)
```
[the remark this answers is not in the document]
Comment (skatedplrn) [reply auger_cli/api/cluster_tasks.py]: I need this code to run automated evaluation, now I do it from notebook
We can keep it private and do not expose to user

File: auger_cli/api/experiments.py [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+    return result
+
+def run(client):
+    client.config.delete_session_file()
+
+    project_id = projects.start(client, create_if_not_exist=True)
+
+    experiment = get_or_create(client, project_id)
+
```
Comment (derekflint) [inline auger_cli/api/experiments.py]: I created some example rest calls but look at api reference as well.

https://app.auger.ai/api/v1/docs
https://github.com/deeplearninc/auger-cli/pull/33/files#diff-c1d681950d6cde9754aae1947c2d55feR118

You should create experiment session here and then use task_args as model_settings argument for experiment_session.

Comment (skatedplrn) [reply auger_cli/api/experiments.py]: Who is going to create dataset manifest? And more important who is tracking changes between runs? Now I do it in this task:
is_dataset_changed_task

To create exp_session we need old or new dataset manifest

Comment (skatedplrn) [reply auger_cli/api/experiments.py]: I use trials APi to get latest leaderboard, this is not an issue

**Pull request merged**